.github/workflows/clusterkubevirtadm-test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.18
+        go-version: 1.20
```

Comment:
Can we retrieve the golang version from `go.mod` with https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file so we don't have to harcode it here.